### PR TITLE
Added two new files

### PR DESCRIPTION
**Pull request by hr1shu**
As discussed in the issues section.
Thanks🖤

### CODE REVIEW

File: python/maths/Factorial.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    return 1
+  return num * factorial(num-1)
+
+def fact(num):
+  result = 1
+  for i in range(1,num+1):
+    result = result * i
+  return result
+
+print(fact(6))
```
Comment (geekquad) [inline python/maths/Factorial.py:12]: Please make it a user-driven program where input is given by the user.

File: python/string algorithms/Reversing a string using Recursion.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def reverse(word):
+  size = len(word)
+  if size == 0 :
+    return 
+  last_char = word[size-1]
+  print(last_char,end='')
+  return reverse(word[0:size-1])
+  
+reverse('hello there')
```
Comment (geekquad) [inline python/string algorithms/Reversing a string using Recursion.py:9]: Please make it a user-driven program where input is given by the user.

**Review by geekquad — CHANGES_REQUESTED** [review]
Hey @hr1shu, great work man. Just make a few changes as mentioned. Thankyou :)

**Comment by hr1shu** [comment]
Please review my codes again and instruct me further.
Thanks.

**Review by geekquad — APPROVED** [review]
Good Job mate!

**Pull request merged**